### ISSUE TITLE
Email Template Design - "Built for you: New in Reportstream" 

### ISSUE THREAD

**Issue by RakshindaAslam** [CDCgov/prime-reportstream#5387]
## Problem statement

As a user, I want to receive emails every release that lists out recently released features and bug fixes in ReportStream in user-friendly format and language that is easy to understand.

## What you need to know
- [ ] [Example 1](https://www.intercom.com/changes/en) and [Example 2](https://mailchimp.com/whats-new/)
- [ ] We currently use SendGrid for sending mass emails.

## Acceptance criteria

- [ ] Same style and format as the rest of the ReportStream site
- [ ] Easy to navigate

## To do

- [ ] ..

## Considerations
- [ ] How will the contents be added to the email?
- [ ] Refer #5118

**Comment by drew-usds** [comment]
From design sync 5/26: 

Getting a full process set up to be automated is a fair amount of work we need to plan out ... this message can just be plain text/sent in outlook for the first couple of updates.

To-do for automated sending:
- Update existing sendgrid templates to work with this content
- Sendgrid config for this new message type
- Unsubscribe process outlined and setup

**Comment by BerniXiongA6** [comment]
@HeatherRienks we noticed this is still open and O&O is also tagged. Looks like this work is done. Can you confirm and feel free to close if that's true. cc: @brandonnava

**Comment by HeatherRienks** [comment]
@BerniXiongA6 & @brandonnava - I missed this comment! My apologies!
Yes, closing this ticket as I believe we (Experience and O&O) decided that we would not send out regular emails to our users. There was a lot of discussion about this - see "Considerations" noted in ticket #5389.

I think the thought was that we did not want to spam our users. Additionally, we rolled out a new process for keeping release notes up to date (as of October 2022), so this can be closed for now.
